docker/php/Dockerfile

```diff
@@ -1,3 +1,8 @@
 FROM php:7-fpm
 
-WORKDIR /quarto
\ No newline at end of file
+RUN apt-get update && apt-get install -y \
```

Comment:
As we discussed, there were missing extensions into the php container to install the project. 
I also added `composer` because the `makefile` install recipe does not install the dev dependencies. So when launching the tests, php needs `composer` to install this dev dep. (phpUnit & Co).